_content/articles/summary-of-mind-and-matter_saddhatissa-hammalawa.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "The Summary of Mind and Matter"
+authors:
+  - "Hammalawa Saddhatisa"
+external_url: "https://palitextsociety.org/wp-content/uploads/2022/01/JPTS_1987_XI.pdf"
```

Comment:
That's a fine source_url. Here's the proper external_url: https://archive.org/download/jpts-xi-1987/Namarupasamaso_%20Summary%20of%20Mind%20and%20Matter%20-%20H%20Saddhatissa_text.pdf